packages/slice-machine/lib/builders/SliceBuilder/FieldZones/index.tsx

```diff
@@ -154,13 +180,20 @@ const FieldZones: FC = () => {
     flushSync(() => setSlice(newSlice));
   };
 
+  const onDeleteRepeatableZone = () => {
+    const newSlice = deleteRepeatableZone({ slice, variationId: variation.id });
+
+    setSlice(newSlice);
+    setIsDeleteRepeatableZoneDialogOpen(false);
+  };
+
   return (
     <List>
       <Zone
         zoneType="slice"
         zoneTypeFormat={undefined}
         tabId={undefined}
-        title="Non-Repeatable Zone"
+        title="Fields"
```

Comment:
With Lucie proposal change with the usage of `primary` I'm wondering if here it could be good to indicate that it's corresponding to primary. Could help new users to understand. WDYT? 

Like `Fields (primary)` 

But maybe it's not interesting, if you like it, we could ask Sam. If you don't, let's not, I'm not super confident on this, mostly wanted to share.

Reply:
I think "Fields" is a better option to keep it consistent with slices that don't have the Repeatable Zone. I think there will be less confusion around why the configuration is different between slices that have and don't have a Repeatable Zone.

Alternatively, we could continue to use "Static Zone" and "Repeatable Zone" only when a Repeatable Zone is active. We would still revert to "Fields" when it is the only zone.

I asked Sam here: https://prismic-team.slack.com/archives/C02L3FN3AJK/p1712869154256539